amqpstorm/io.py

```diff
@@ -272,6 +272,9 @@ def _receive(self):
             data_in = self._read_from_socket()
         except socket.timeout:
             pass
+        except ssl.SSLWantReadError:
+            # Retry if the non-blocking socket does not have any meaningful data ready.
```

Comment:
This line is a little long. Break the comment into two, and optionally turn it into a `note`.

```
# NOTE(visobet): Retry if the non-blocking socket does not
# have any meaningful data ready.
```